File: Android/data/pool/src/main/java/com/felipearpa/tyche/data/pool/domain/PoolGamblerScoreResponseTransformer.kt
package com.felipearpa.tyche.data.pool.domain

internal fun PoolGamblerScoreResponse.toPoolGamblerScore() =
    PoolGamblerScore(
        poolId = poolId,
        poolName = poolName,
        gamblerId = gamblerId,
        gamblerUsername = gamblerUsername,
        position = position,
        beforePosition = beforePosition,
        score = score,
        gamblerCount = gamblerCount,
    )

File: iOS/DataPool/Sources/DataPool/Domain/PoolGamblerScoreResponseTransformer.swift
extension PoolGamblerScoreResponse {
    func toPoolGamblerScore() -> PoolGamblerScore {
        return PoolGamblerScore(
            poolId: poolId,
            poolName: poolName,
            gamblerId: gamblerId,
            gamblerUsername: gamblerUsername,
            position: position,
            beforePosition: beforePosition,
            score: score,
            gamblerCount: gamblerCount
        )
    }
}
